main.rb

```diff
@@ -0,0 +1,39 @@
+require './app'
+
+def main
+  app = App.new
+  app.start_console
+end
```

Comment:
- [ ] When I try to run your project using `ruby main.rb`, I get the following error, kindly fix this issue and make sure your project runs properly and all features for basic UI milestone works correctly
>![image](https://user-images.githubusercontent.com/53402600/216764605-4568f488-90cb-4563-a2f5-2b74c36b8262.png)